SPECS/perl-Parse-PMFile/perl-Parse-PMFile.spec

```diff
@@ -0,0 +1,153 @@
+Name:           perl-Parse-PMFile
+Version:        0.42
+Release:        4%{?dist}
+Summary:        Parses .pm file as PAUSE does
+License:        GPL+ OR Artistic
+Group:          Development/Libraries
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://metacpan.org/release/Parse-PMFile
+Source0:        https://cpan.metacpan.org/authors/id/I/IS/ISHIGAKI/Parse-PMFile-%{version}.tar.gz
+# Remove useless dependency on ExtUtils::MakeMaker::CPANfile
+Patch0:         Parse-PMFile-0.41-Do-not-use-ExtUtils-MakeMaker-CPANfile.patch
+BuildArch:      noarch
+BuildRequires:  make
+BuildRequires:  perl
+BuildRequires:  perl(ExtUtils::MakeMaker) >= 6.76
+BuildRequires:  perl(strict)
+BuildRequires:  perl(warnings)
+# Run-time
+BuildRequires:  perl(Dumpvalue)
+BuildRequires:  perl(File::Spec)
+BuildRequires:  perl(JSON::PP) >= 2.00
+BuildRequires:  perl(Safe)
+BuildRequires:  perl(version) >= 0.83
+# Tests
```

Comment:
If these are test dependencies, can we gate their inclusion at build-time behind the %with_check variable we set during check builds?

Reply:
Added %with_check for tests.